view/frontend/web/js/view/payment/method-renderer/billink.js

```diff
@@ -210,6 +208,8 @@ define(
 
                     quote.billingAddress.subscribe(
                         function(newAddress) {
+                            this.businessMethod = quote.billingAddress().company !== null ? BUSINESS_METHOD_B2B : BUSINESS_METHOD_B2C;
```

Comment:
not sure it will be always null (e.g.: empty string)
so maybe just quote.billingAddress().company ? BUSINESS_METHOD_B2B : BUSINESS_METHOD_B2C

Reply:
fixed